main.go

```diff
@@ -12,6 +13,20 @@ import (
 	"github.com/robbiemcmichael/paxy/pkg/proxy"
 )
 
+type Server struct {
+	Proxy  *proxy.Proxy
+	Source []byte
+}
+
+func (server *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if r.Method == http.MethodGet && r.URL.Host == "" && r.URL.Path == "/pac" {
+		w.Write([]byte(server.Source))
+		w.Write([]byte("\n"))
```

Comment:
What's the extra newline character for? Shouldn't we reproduce exactly the same bytes that we read from the file?

Reply:
Hmm, for some reason I thought it wasn't sending the last line without it, but I double checked this morning and it works fine, so will remove.